cold-wallet/wallet/cold-wallet.ts

```diff
@@ -46,6 +52,18 @@ export class ColdWallet {
     return this.wallet.get(index).getAddress();
   }
 
+  public async getAddresses(count: number): Promise<OwnerAddress[]> {
```

Comment:
shouldn't we add a start index? and maybe we shouldn't start at 0? because that will be the LIQ address... or are we manually removing LIQ address?

Reply:
Start index could make sense, will add it. This whole utility was just to recover the owner list, in case we lost it 🤣 I would therefore include index 0, as we also need to know the liquidity address.